src/core/api/health.go

```diff
@@ -260,22 +260,10 @@ func databaseHealthChecker() health.Checker {
 }
 
 func redisHealthChecker() health.Checker {
-	url := config.GetRedisOfRegURL()
-	timeout := 60 * time.Second
 	period := 10 * time.Second
 	checker := health.CheckFunc(func() error {
-		conn, err := redis.DialURL(url,
-			redis.DialConnectTimeout(timeout*time.Second),
-			redis.DialReadTimeout(timeout*time.Second),
-			redis.DialWriteTimeout(timeout*time.Second))
-		if err != nil {
-			return fmt.Errorf("failed to establish connection with Redis: %v", err)
-		}
+		conn := util.DefaultPool().Get()
```

Comment:
No need to fire a `PING`?

Reply:
no need manually ping, the pool borrow will do it auto